plugins/vst_base/CMakeLists.txt

```diff
@@ -39,23 +39,43 @@ IF(LMMS_HOST_X86_64)
 	IF("${WINEBUILD_OUTPUT}" MATCHES ".*x86_64-linux-gnu/wine/libwinecrt0.a.*")
 		SET(EXTRA_FLAGS ${EXTRA_FLAGS} -nodefaultlibs /usr/lib/i386-linux-gnu/wine/libwinecrt0.a -luser32 -lkernel32 -lgdi32)
 	ENDIF()
-	#The following check works on Fedora systems 
-	IF("${WINEBUILD_OUTPUT}" MATCHES ".*lib64/wine/libwinecrt0.a.*")
+	# The following check works on Fedora systems
+	IF("${WINEBUILD_OUTPUT}" MATCHES "/usr/lib/lib64/wine/libwinecrt0.a.*")
 		SET(EXTRA_FLAGS ${EXTRA_FLAGS} -nodefaultlibs /usr/lib/i386/wine/libwinecrt0.a -luser32 -lkernel32 -lgdi32)
 	ENDIF()
+	# Development
+	IF("${WINEBUILD_OUTPUT}" MATCHES "/opt/wine-devel/lib64/wine/libwinecrt0.a.*")
```

Comment:
I'm confused by this comment. What I did is just added a 2 paths to look for libwinecrt0.a.
In addition to /usr/lib/x86_64-linux-gnu/wine/ and /usr/lib/lib64/wine/ it now looks also into /opt/wine-devel/lib64/wine/ and /opt/wine-staging/lib64/wine/. I didn't introduce any new mixing of 32bit and 64bit.